be/test/exprs/vectorized/bitmap_functions_test.cpp

```diff
@@ -1944,4 +1944,216 @@ TEST_F(VecBitmapFunctionsTest, array_to_bitmap_test) {
     ASSERT_EQ(res->debug_item(0), "");
 }
 
+TEST_F(VecBitmapFunctionsTest, sub_bitmap) {
+    BitmapValue bitmap({1, 2, 3, 4, 5, 64, 128, 256, 512, 1024});
+    auto bitmap_column = BitmapColumn::create();
+    bitmap_column->append(&bitmap);
+
+    {
+        Columns columns;
+        auto offset = Int64Column::create();
+        auto len = Int64Column::create();
+        offset->append(2);
+        len->append(3);
```

Comment:
what if offset and/or len is null?

Reply:
Thanks @fzhedu for your review. Do you mean to add test cast for `offset and/or len is null`?